src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/feature_value.yml

```
@@ -0,0 +1,25 @@
+admin_feature_values_index:
+  path: /
+  methods: [ GET ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\FeatureValueController::indexAction
+    _legacy_controller: AdminFeatures
+    _legacy_link: AdminFeatures
```

I think the legacy controller also depended on a feature_id parameter so you can also add:
```suggestion
    _legacy_link: AdminFeatures
    _legacy_parameters:
      id_feature: featureId
```

---

I see that you added the parameter in the prefix `prefix: /features/{featureId}/values/` it's a bit confusing because we don't have all the elements readable in this route definition

I think you should move these routes into the `features.yml` file instead (they are tightly coupled anyway so no need to split them). It already has the `/features` prefix so this way you can set the path for this route as `/{featureId}/values/`